deploy/general-deploy/README.md

```diff
@@ -27,7 +27,7 @@ In order to connect to your Turbonomic installation, a Kubeturbo configuration f
 Create a file called `config` in the `/etc/kubeturbo/` directory in the same node labeled in previous step, with the following contents:
 
 > The `<TURBONOMIC_SERVER_URL>` is typically `https://<TURBO_SERVER_IP>:443`
-> The `<TURBONOMIC_SERVER_VERSION>` is Turbonomic release version, e.g. `5.9.0` or `6.0.0`
+> The `<TURBONOMIC_SERVER_VERSION>` is Turbonomic release version, e.g. `5.9` or `6.0`
```

Comment:
Isn't this the server version? Shouldn't that still be 6.0.0?

Reply:
yea, you are right. I will change them back.